src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs

```diff
@@ -283,4 +283,31 @@ public void SortedSet_Generic_CopyTo_NegativeCount_ThrowsArgumentOutOfRangeExcep
 
         #endregion
     }
+
+    public class SortedSet_ICollection_NonGeneric_Tests : ICollection_NonGeneric_Tests
+    {
+        protected override bool ICollection_NonGeneric_CopyTo_ArrayOfEnumType_ThrowsArgumentException { get { return true; } }
+        protected override bool Enumerator_Current_UndefinedOperation_Throws { get { return true; } }
+
+        protected override void AddToCollection(ICollection collection, int numberOfItemsToAdd)
+        {
+            int seed = numberOfItemsToAdd * 34;
```

Comment:
I could replace all seed instantiations with a set to some base const seed if you'd prefer. I used random values because it introduced some different values to the methods without sacrificing the deterministic quality, but I see that they're somewhat messy.

I guess I could also make a method that returns a seed based on the method name. That would still allow different seed values while retaining determinism and cleanliness.